github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -8,6 +8,7 @@ presubmits:
       testgrid-dashboards: kubevirt-presubmits
     always_run: false
     optional: true
+    cluster: prow-workloads
```

Comment:
Please mention that we set this on the new cluster in the commit message.

Reply:
Done

Reply:
New cluster will xfail the two tests of sig-network that use the docker bridge,

also it wont work still on the new cluster, (some labels should be changed) unless @fgimenez  already fixed it

Reply:
Not yet, will let you all know when ready.

Reply:
Thanks, but meanwhile is it fine to `cluster: prow-workloads` and putting the 3 labels as false right ?
As this what was done in this PR
Following your tip here https://github.com/kubevirt/project-infra/pull/1081#issuecomment-813964129